cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -1410,10 +1410,11 @@ public String findBash() {
 
     String bash = BASH;
     if (SystemInfoImpl.INSTANCE.isWindows()) {
-      bash = findBashOnWindows();
+      String variable = IdeVariables.BASH_PATH.getName();
+      bash = getVariables().get(variable);
```

Comment:
Create `Path` and check if exists.
Then do debug logging if exists and warning if variable is defined but points to non existing file.